dockerfiles/che-fabric8/assets/branding/product.json

```diff
@@ -5,12 +5,6 @@
     "faq": "https://www.eclipse.org/che/docs/che-7/hosted-che/#hosted-che-faq-and-troubleshooting_hosted-che"
   },
   "configuration": {
-    "menu": {
-      "disabled": ["organizations", "factories"]
-    }, 
-    "features": {
-      "disabled": ["workspaceSharing", "kubernetesNamespaceSelector"]
```

Comment:
@ibuziuk Do you see any issue if new dashboard will show readonly namespace on Create Workspace page?
![Screenshot_20210114_165650](https://user-images.githubusercontent.com/5887312/104607474-895cca00-5689-11eb-93f0-9422ac016b91.png)
![Screenshot_20210114_165659](https://user-images.githubusercontent.com/5887312/104607479-8a8df700-5689-11eb-9d6a-b21c74f11b11.png)

Reply:
this is fine I believe